hs_core/search_indexes.py

```diff
@@ -171,65 +177,67 @@ class BaseResourceIndex(indexes.SearchIndex, indexes.Indexable):
     # viewer = indexes.MultiValueField(faceted=True)
     # viewers_count = indexes.IntegerField(faceted=True)
     # editor_login = indexes.MultiValueField(faceted=True)
     # editor = indexes.MultiValueField(faceted=True)
     # editors_count = indexes.IntegerField(faceted=True)
-    person = indexes.MultiValueField(faceted=True)
+    person = indexes.MultiValueField(stored=False)
 
     # non-core metadata
-    geometry_type = indexes.CharField(faceted=True)
-    field_name = indexes.CharField()
-    field_type = indexes.CharField()
-    field_type_code = indexes.CharField()
-    variable = indexes.MultiValueField(faceted=True)
-    variable_type = indexes.MultiValueField(faceted=True)
-    variable_shape = indexes.MultiValueField()
-    variable_descriptive_name = indexes.MultiValueField()
-    variable_speciation = indexes.MultiValueField()
-    site = indexes.MultiValueField()
-    method = indexes.MultiValueField()
-    quality_level = indexes.MultiValueField()
-    data_source = indexes.MultiValueField()
-    sample_medium = indexes.MultiValueField(faceted=True)
-    units = indexes.MultiValueField(faceted=True)
-    units_type = indexes.MultiValueField(faceted=True)
-    aggregation_statistics = indexes.MultiValueField()
+    geometry_type = indexes.CharField(stored=False)
+    field_name = indexes.CharField(stored=False)
+    field_type = indexes.CharField(stored=False)
+    field_type_code = indexes.CharField(stored=False)
+    variable = indexes.MultiValueField(stored=False)
+    variable_type = indexes.MultiValueField(stored=False)
+    variable_shape = indexes.MultiValueField(stored=False)
+    variable_descriptive_name = indexes.MultiValueField(stored=False)
+    variable_speciation = indexes.MultiValueField(stored=False)
+    site = indexes.MultiValueField(stored=False)
+    method = indexes.MultiValueField(stored=False)
+    quality_level = indexes.MultiValueField(stored=False)
+    data_source = indexes.MultiValueField(stored=False)
+    sample_medium = indexes.MultiValueField(stored=False)
+    units = indexes.MultiValueField(stored=False)
+    units_type = indexes.MultiValueField(stored=False)
+    # unused # aggregation_statistics = indexes.MultiValueField(stored=False)
     absolute_url = indexes.CharField(indexed=False)
 
     # extra metadata
-    extra = indexes.MultiValueField()
+    extra = indexes.MultiValueField(stored=False)
+
+    # json field for discovery
+    # json = indexes.CharField(use_template=True, indexed=False)
 
     def get_model(self):
         """Return BaseResource model."""
         return BaseResource
 
     def index_queryset(self, using=None):
         """Return queryset including discoverable and public resources."""
-        return self.get_model().objects.filter(Q(raccess__discoverable=True) |
-                                               Q(raccess__public=True)).distinct()
+        return self.get_model().objects.filter(Q(raccess__discoverable=True)).distinct()
```

Comment:
I see the pubic or discoverable checks being reduced to discoverable in several places.  Is this because a public resource is always discoverable?  Where is the rule that enforces this?

Reply:
The two-stage mechanism was before I made published --> public --> discoverable. They're redundant now.